doc/make.jl

```diff
@@ -4,6 +4,9 @@ Pkg.init()
 cp(joinpath(@__DIR__, "REQUIRE"), Pkg.dir("REQUIRE"); remove_destination = true)
 Pkg.update()
 Pkg.resolve()
+Pkg.checkout("Documenter")
+Pkg.checkout("Compat")
+Pkg.checkout("Compat", "fe/linalg") # to make Compat load
```

Comment:
Artifact that should eventually disappear? Marking in case so :).

Reply:
Well, this PR breaks Compat, and we can not set the  `VERSION` check here https://github.com/JuliaLang/Compat.jl/blob/fa6fb2db42a68990f28014ff1290259d36124075/src/Compat.jl#L799-L803 before this PR is merged :) So if it is OK we leave it like this and then when this is merged we can add the version to Compat and remove this? :slightly_smiling_face: